content/rs/administering/designing-production/networking/port-configurations.md

```diff
@@ -51,3 +51,12 @@ To change the default port for the RS Web UI, on any node in the cluster run:
 ```src
 rladmin cluster config cm_port <new-port>
 ```
+
+## Turning off HTTP support
+
+Some of our customers want to be able to harden our web UI by turning off HTTP support by default. This will ensure that no traffic will be sent to an unencrypted port, forcing encryption in transit via HTTP.
```

Comment:
"no traffic"- I think it's too general...

Reply:
@Paz-Yanover - I made this a bit more specific with my last two commits. Better?